src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private Random random = new Random();
+
+    public Figure getRandomFigure() {
+
+        Figure randomFigure;
+
+        int randomType = random.nextInt(5 - 1 + 1) + 1;
+        switch (randomType) {
+            case 1 -> randomFigure = new Circle();
+            case 2 -> randomFigure = new Rectangle();
+            case 3 -> randomFigure = new Square();
+            case 4 -> randomFigure = new IsoscelesTrapezoid();
+            case 5 -> randomFigure = new RightTriangle();
+            default -> throw new IllegalStateException();
```

Comment:
Return default figure instead of illegal state
```suggestion
            default -> throw new IllegalStateException();
```